nemo/collections/tts/torch/data.py

```diff
@@ -465,7 +476,9 @@ def __getitem__(self, index):
                     fill_na=0.0,
                 )
                 pitch = torch.from_numpy(pitch).float()
-                torch.save(pitch, pitch_path)
+                voiced_mask = torch.from_numpy(voiced_mask).float()
+                p_voiced = torch.from_numpy(p_voiced).float()
+                torch.save({'pitch': pitch, 'voiced_mask': voiced_mask, 'p_voiced': p_voiced}, pitch_path)
```

Comment:
Let's not do this. Can you save p_voiced and voiced_mask to separate files? And only if the user requests it when they init TTSDataset

Reply:
Done! I am now saving them separately. Also user has to request in the init TTSDataset